app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<!-- Main page -->
+
+<section class="wrapper">
+  <%= render partial: "form_index" %>
```

Comment:
Since we're only using the `form_index` partial in this location, it probably does need to be a partial.